server/commands.go

```diff
@@ -274,6 +276,20 @@ func NewServerCommands(defaultCfg *sconfig.Config) []*cli.Command {
 				}
 				if c.String(common.FlagDBPath) == "" {
 					opts = append(opts, WithPersistenceDisabled())
+
+					if clusterCfg, err := cliconfig.NewConfig("temporalio", "temporal-version-info"); err == nil {
```

Comment:
Please add some comments in this block to explain that we're explicitly ignoring errors.